src/storage/mod.rs

```diff
@@ -4256,13 +4256,16 @@ mod tests {
         let storage = TestStorageBuilder::new().build().unwrap();
         let (tx, rx) = channel();
 
+        let k = Key::from_raw(b"k");
+        let v = b"v".to_vec();
+
         // No lock.
         storage
             .async_txn_heart_beat(
                 Context::default(),
-                Key::from_raw(b"k"),
-                10,
-                100,
+                k.clone(),
+                /*start_ts*/ 10,
```

Comment:
I remember block comments are not idiomatic in Rust...

Reply:
I think I can add a description on each invoke of the function instead commenting arguments..